.gitignore

```diff
@@ -4,7 +4,7 @@
 # Generated by Cargo
 # will have compiled files and executables
 debug/
-target/
+target*
```

Comment:
Feels like it maybe belongs in a personal `.gitignore`. Can we at least change to `target-alpine/`? Otherwise I worry this will cause a confusing behavior in the future when we forget about it.